src/react_native/utils.cljs

```diff
@@ -0,0 +1,20 @@
+(ns react-native.utils)
+
+(defonce throttle (atom {}))
```

Comment:
We can make this global state private. Just so we discourage anyone from playing with fire.